Makefile

```diff
@@ -160,7 +160,7 @@ override refs = \
 	Mono.Debugging.Soft.dll
 
 $(addprefix bin/, $(refs)):
-	$(CD) dep/debugger-libs && $(XBUILD) $(XBUILD_FLAGS) debugger-libs.sln
+	$(CD) dep/debugger-libs && nuget restore && $(XBUILD) $(XBUILD_FLAGS) debugger-libs.sln
```

Comment:
Please add a `NUGET` variable at the top of the file similar to the others, and use that here. Also document it in `README.md`.